nntrainer/models/neuralnet.cpp

```diff
@@ -782,11 +784,11 @@ int NeuralNetwork::train_run(std::function<bool(void *userdata)> stop_cb) {
 
   auto train_for_iteration = [this, stop_cb](RunStats &stat,
                                              DataBuffer &buffer) {
-    model_graph.flushCache();
-
     forwarding(true, stop_cb);
     backwarding(iter++, stop_cb);
 
+    model_graph.flushCache();
```

Comment:
Is there any reason to place this here? if it is, then please leave the comments.

Reply:
thanks. a comment is leaved.